lib/vyasa_web/components/contexts/read.ex

```diff
@@ -1262,41 +1422,48 @@ defmodule VyasaWeb.Context.Read do
   # TODO: sheaf-crud: reply_to is currently set to the same as the active_sheaf
   def render(assigns) do
     ~H"""
-    <div id={@id} class="flex-grow" >
+    <div id={@id} class="flex-grow">
       <!-- CONTENT DISPLAY: -->
       <div id="content-display" class="mx-auto max-w-2xl">
-          <.live_component
-            :if={@content_action == :show_sources}
-            module={VyasaWeb.Context.Read.Sources}
-            id="content-sources"
-            sources={@streams.sources}
-            user_mode={@user_mode}
-          />
-
-          <.live_component
-            :if={@content_action == :show_chapters}
-            module={VyasaWeb.Context.Read.Chapters}
-            id="content-chapters"
-            source={@source}
-            chapters={@streams.chapters}
-            user_mode={@user_mode}
-          />
-
-          <.live_component
-            :if={@content_action == :show_tracklists}
-            module={VyasaWeb.Context.Read.Tracklists}
-            id="content-tracklists"
-            tracklists={@streams.trackls}
-            user_mode={@user_mode}
-          />
-
-         <.live_component
-            :if={@content_action == :show_tracks}
-            module={VyasaWeb.Context.Read.Tracks}
-            id="content-tracks"
-            tracks={@streams.tracks}
-            user_mode={@user_mode}
-          />
+        <.debug_dump
+          label="Read Mode State Checks"
+          content_action={@content_action}
+          tracklist_cursor={@tracklist_cursor}
+          tracklist_id={@tracklist_id}
```

Comment:
this visual debug dump is intended to help with ensuring correctness of the first mount state. 

previously, the state issues @ mount that we experienced were happening because of the way that the injection of tracklist id was being trackloaded. 

this is expected. 

the integration task is what completes the proper wire-up of this.